.github/pull_request_template.md

```diff
@@ -1,12 +1,25 @@
-<!--
-- Recipes should be `.md` files in the `src/` directory.  Look at already
-  existing `.md` files for examples or see [example](example.md).
-- File names should be the name of the dish with words separated by hyphens
-  (`-`). Not underscores, and definitely not spaces.
-- Recipe must be based, i.e. good traditional and substantial food. Nothing
-  ironic, meme-tier hyper-sugary, meat-substitute, etc.
-- Be sure to add a small number of relevant  tags to your recipe (and remove
-  the dummy tags). Try to use already existing tags.
-- Don't include salt and pepper and other ubiquitous things in the ingredients
+### Adding a Recipe Guidelines
+
+*Before you press submit please ensure that you've read and understood both the [readme](#readme.md) and this section.*
+
+*By submitting this pull request I confirm I've read and complied with the below requirements and I realise that if I do not, my pull request will be closed and I will have to resubmit.*
+
+*I also understand that I waive any pretense of ownership to the provided information submitted to this repository.*
+
+- [ ] *I understand that I've read the following.*
+
+### Ensure that
+
+- Recipes should be `.md` files placed into the `src/` directory. Look at already existing `.md` files for examples or see the [example template](example.md).
+- File names should be the name of the dish with any spaced words separated by hyphens.
+  - ✅ `dish-name.md`
+  - ❌ `dish_name.md`
+  - ❌ `dish name.md`
+- Be sure to add a small number of relevant  tags to your recipe (and remove the dummy tags). Try to use already existing tags.
+  - ✅ `;tags: side american`
+  - ❌ `;tags: tag1 tag2 tag3`
+  - ❌ `;tags: tag1 tag2 tag3 side american`
+  - ❌ `;tags: side, american`
+  
```

Comment:
Inconsistent spacing.

Reply:
@Knarkzel is the spacing in the tags? or elsewhere

Reply:
Resolved in latest push.